### ISSUE TITLE
The EPE of LightStereo-S on the KITTI12 test set is 3.3.

### ISSUE THREAD

**Issue by 408550969** [XiandaGuo/OpenStereo#195]
Hello, I used the LightStereo-S model pre-trained on SceneFlow from the Model Zoo as a pre-trained model, trained it using the default lightstereo_s_kitti.yaml, and after completing 500 iterations, I tested it using kitti12_eval.yaml. The results are as follows:
Epoch 0 metrics: {'d1_all': tensor(18.5672), 'epe': tensor(3.3953), 'thres_1': tensor(51.3214), 'thres_2': tensor(27.9768), 'thres_3': tensor(19.1834)}
I remember that in the paper, LightStereo-S achieved an EPE of 0.6. Why is the EPE so high in my training results? Even when I used other mixed data for pre-training and then fine-tuned on KITTI, the EPE remained high.

**Comment by 408550969** [comment]
Is there a training log available (not TensorBoard)?

**Comment by XiandaGuo** [comment]
For the KITTI fine-tuning experiments, we pre-trained on the SceneFlow dataset using different data augmentation strategies. We have updated the configuration; please refer to https://github.com/XiandaGuo/OpenStereo/pull/196.
